docs/install/install-server-with-docker/install-server.inc

```diff
@@ -21,7 +21,7 @@ Here is a minimalist docker-compose file content that can be used to deploy the
     version: '3'
     services:
         postgres:
-            container_name: iso_postgres
+            container_name: oss_postgres
```

Comment:
I might be wrong but I believe this would require a change at line 41 as well.

Reply:
Ah indeed, and the default config of the server as well (which is in the inmanta repo and also requires a rebuild of all the existing container images)